### ISSUE TITLE
ja_ginza_dict-3.1.0-1.tar.gz has inconsistent version: filename has '3.1.0.post1', but metadata has '3.1.0'

### ISSUE THREAD

**Issue by BorysVrublevskyi** [megagonlabs/ginza#158]
Can't install ja_ginza_dict from https://pypi.org/project/ja-ginza-dict/#files with regular `pip install ja-ginza-dict`

```
WARNING: Discarding https://files.pythonhosted.org/packages/e8/e5/c4ea3f165acb38fa125e992d1a6fcfb9a5a8ab6a14cc5010c836713c386b/ja_ginza_dict-3.1.0-1.tar.gz#sha256=f134ca9cf2e41590f7bcd13b7a25d42163cd3d414fc1a53661a9148efb9a3d2d (from https://pypi.org/simple/ja-ginza-dict/). Requested ja-ginza-dict from https://files.pythonhosted.org/packages/e8/e5/c4ea3f165acb38fa125e992d1a6fcfb9a5a8ab6a14cc5010c836713c386b/ja_ginza_dict-3.1.0-1.tar.gz#sha256=f134ca9cf2e41590f7bcd13b7a25d42163cd3d414fc1a53661a9148efb9a3d2d has inconsistent version: filename has '3.1.0.post1', but metadata has '3.1.0'
ERROR: Could not find a version that satisfies the requirement ja-ginza-dict
ERROR: No matching distribution found for ja-ginza-dict
```

But if I will download and change filename from `ja_ginza_dict-3.1.0-1.tar.gz` to `ja_ginza_dict-3.1.0.tar.gz` it will install successfully: `pip install ja_ginza_dict-3.1.0.tar.gz`

Windows Server 2019, Python 3.6.5, Pip 21.0.1

**Comment by hiroshi-matsuda-rit** [comment]
In fact, there is no need to install ja_ginza_dict package for ginza v4.x.
Which version are you using? @BorysVrublevskyi

**Comment by tomgun132** [comment]
@hiroshi-matsuda-rit sorry if I ask here, but I have the same problem as Borys with error 
```ERROR: Could not find a version that satisfies the requirement ja-ginza-dict
ERROR: No matching distribution found for ja-ginza-dict```

My ginza version is 3.1.2. I tried updating to the recent version 4.x, but my classifier model that uses ginza as the embedding has lower accuracy than the version 3.1.2.

**Comment by BorysVrublevskyi** [comment]
Solved by downgrade pip to v20.3.4
